packages/shared-components/src/components/onto-tooltip/onto-tooltip.tsx

```diff
@@ -0,0 +1,102 @@
+import {Component, Element, h, Host, Prop, Watch} from '@stencil/core';
+import tippy, {Instance, Placement} from 'tippy.js';
+import {OntoTooltipConfiguration} from './models/onto-tooltip-configuration';
+
+@Component({
+  tag: 'onto-tooltip',
+  styleUrl: 'onto-tooltip.scss',
+  shadow: false,
+})
+export class OntoTooltip {
+  private tooltip: Instance;
+  private showFunction;
+  private hideFunction;
+
+  @Element() el: HTMLElement;
+
+  /**
+   * The content ot tooltip that will be displayed when mouse is over the element.
+   */
+  @Prop() tooltipContent: string;
+
+  /**
+   * Specifies the position of the tooltip.
+   */
+  @Prop() placement: Placement;
+
+  @Watch('tooltipContent')
+  configurationChanged() {
+    this.update();
+  }
+
+  componentDidLoad() {
+    this.init();
+  }
+
+  disconnectedCallback() {
+    this.destroy();
+  }
+
+  render() {
+    return (
+      <Host class="yasgui-tooltip">
+        <slot />
+      </Host>
+    );
+  }
+
+  private update() {
+    this.init();
+    this.tooltip.setContent(this.tooltipContent);
+  }
+
+  private init() {
+    if (this.tooltip) {
+      return;
+    }
+    this.tooltip = tippy(this.el, this.getTooltipOptions());
+
+    this.showFunction = this.createShowFunction(this.tooltip);
```

Comment:
And now we reach to the point where we need to have these named consistently as we do it everywhere else. For example, we never name event handlers like `someFunction`... Rename these to `onMouseOver` and `onMouseLeave`